.github/workflows/push-foundation-repo.yml

```diff
@@ -1,4 +1,4 @@
-name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo.
```

Comment:
Nit: All the other names don't have punctuation, please make this consistent (whichever way you prefer).